saos-webapp/src/main/webapp/WEB-INF/view/details/judgmentDetailsContent.jsp

```diff
@@ -3,7 +3,7 @@
 
 
 <%-- Judgment content --%>            
-<h1 ><spring:message code="judgmentDetails.judgmentFullText" /></h1>
+<h2 class="content-header" ><spring:message code="judgmentDetails.judgmentFullText" /></h2>
```

Comment:
Hmm, in my view it should be h1 not h2. That's what have been agreed with AkcesLab.